### ISSUE TITLE
Add Panel - check id for uniquness

### ISSUE THREAD

**Issue by boldtrn** [nickpeihl/leaflet-sidebar-v2#29]
Currently, when using the addPanel method using an already existing id, the panels get added on top of each other, which is not ideal. Maybe it would make sense to add an overwrite flag to the method or show a warning if the panel already exists?

**Comment by noerw** [comment]
Good point. We could think of a mechanism to update a panel, but this should then be in a separate function `updatePanel(id, update)`.
So `addPanel()` should throw an error for a duplicate ID.
